docs/tutorials/java/mxnet_java_on_intellij.md

```diff
@@ -86,18 +86,18 @@ Set the project's location. The rest of the settings can be left as their defaul
 After clicking Finish, you will be presented with the project's first view.
 The project's `pom.xml` will be open for editing.
 
-**Step 3.** Add the following Maven dependency to your `pom.xml` file under the `dependencies` tag: 
+**Step 3.** Download the MXNet Java API package from [here](https://repository.apache.org/#nexus-search;gav~org.apache.mxnet~~1.3.1-SNAPSHOT~~) and add the following Maven dependency to your `pom.xml` file under the `dependencies` tag: 
```

Comment:
Why should users download it manually? why not rely on Maven to do it for users?